src/libs/Navigation/linkingConfig.js

```diff
@@ -73,6 +73,10 @@ export default {
                                 path: ROUTES.SETTINGS_WALLET,
                                 exact: true,
                             },
+                            Settings_New_Wallet: {
```

Comment:
Update the `Settings_Wallet` screen above to reference `SCREENS.SETTINGS.WALLET` instead of being hard-coded

Reply:
Done!